app/src/main/java/com/epfl/drawyourpath/database/MockDatabase.kt

```diff
@@ -583,13 +632,19 @@ class MockDatabase : Database() {
         return CompletableFuture.completedFuture(Unit)
     }
 
+    override fun getAllTournamentsId(): LiveData<List<String>> {
+        return MOCK_TOURNAMENTS_ID
+    }
+
     override fun getTournamentUID(): String {
-        return mockTournamentUID++.toString()
+        return mockUID++.toString()
```

Comment:
It's normal that we used the mock user id ?

Reply:
It's not the userID it's a unique id, I agree that it's a bit confusing. I wil rename UID to UniqueId.